### ISSUE TITLE
Consolidate content on the website, drop the github wiki

### ISSUE THREAD

**Issue by simonmichael** [plaintextaccounting/plaintextaccounting#129]
Hello - I am feeling we should drop the wiki and merge its content back to the static site (keeping the structure much the same, so the site will now be a multi-page site, no longer essentially just a single page); and I propose to try this fairly soon.

Rationale and context: IIRC Felix Van der Jeugt started adding content in the plaintextaccounting.org repo's wiki at the beginning of 2022, and pretty soon I joined in and moved/added content there, and spread the PTA site across the "static site" and the github wiki, with the goal of easier content management by the community and more room to grow on the wiki side.

Since then the wiki has received 220 commits by me and 30 from 14 other committers. That was good, but at an average of 0.6 new contributors and 1.3 contributed commits per month over two years I feel the wiki is not justifying the added complexity of having the site split in two. I also am feeling it's wise to be reducing dependence on Github rather than increasing it. And I feel having everything in our own site gives greater potential for customisation and future enhancements.

**Comment by colindean** [comment]
Sounds good to me.

**Comment by simonmichael** [comment]
Good progress on this, but I've had trouble with deploying on Cloudflare. Cloudflare Pages V1's pandoc doesn't support the wikilinks extension I'm using, and Cloudflare Pages V2 doesn't have pandoc. I figured out how to install it temporarily, and the site seems to be deploying successfully, and I have purged its Cloudflare cache, but still no visible change. Perhaps it will self-correct soon. Meanwhile I took the wiki down (too soon) and I'm out of time for right now. Will fix soon, hopefully this evening.
